v21.1/cockroach-userfile-upload.md

```diff
@@ -125,6 +125,126 @@ cockroach userfile upload /Users/maxroach/Desktop/test-data.csv userfile://testd
 successfully uploaded to userfile://testdb.public.uploads/test-data.csv
 ~~~
 
+### Upload a backup directory recursively
+
+Currently in v21.1 and prior, it's [not possible to use `cockroach userfile upload` recursively](#known-limitation) when uploading a directory of files. When you need to upload a directory (such as a backup) to `userfile` storage, it is possible to programmatically upload your files.
+
+The following example uses `userfile` storage and its features to backup a database and then restore it to a cluster. When quick testing is required, or, to protect against accidental loss of data, this workflow provides a way to recover in those scenarios when using `userfile` as your storage option.
```

Comment:
Several versions of comment, your choice which makes more sense

-----

Minimum comment is:

> or, to protect

unnecessary comma

-----

Medium comment is:

tl;dr: this clause is not really necessary:

> When quick testing is required, or, to protect against accidental loss of data, ...

and the sentence could be: "This workflow provides ..."

The way this is phrased with the mention of accidental data loss up front is a little scary.  Also I'm not sure I fully understand what is meant.  Below, it doesn't seem like I am going to have any data loss using these instructions (at least based on my imperfect understanding).  It looks like I am doing 

CRDB userfile -> my file system -> CRDB userfile -> CRDB SQL RESTORE

which seems pretty straightforward with no data loss things happening (modulo errors etc of course, as always)